app/src/main/java/org/eyeseetea/malariacare/layout/adapters/dashboard/ReviewScreenAdapter.java

```diff
@@ -31,6 +36,23 @@ public ReviewScreenAdapter(List<Value> items, LayoutInflater inflater, Context c
         this.headerLayout = R.layout.review_header;
         this.subHeaderLayout = R.layout.review_sub_header;
         this.recordLayout = R.layout.review_item_row;
+        List<String> colorsList = new ArrayList<>();
```

Comment:
@idelcano For a better readable code, we should extract color list creation to own method